### PR TITLE
To-Do: Interactive

### PR DESCRIPTION
**Pull request by danielochuba**
# To-Do: INTERACTIVE-LIST
- Allows the user to add a Todo task using the ADD Button.
- Remove a Task using the Trash icon.
- edit saved Task
- Clear all checked ✔️ item using the **Clear completed** Button

### CODE REVIEW

File: modules/TaskHandle.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+//     <input type="checkbox" ${todo.completed ? 'checked' : ''} class="checkbox"/>
+//     <input type="text" value="${todo.description}" class="inputtext" id="${todo.id}"/>
+//     <button type="button">🗑</button>
+//   </li>
+// `).join('');
+
+//   todoList.innerHTML = description;
+// }
+/**
+ * Updates the todos list with new items.
```
Comment (VICTOR-CHUKWU) [inline modules/TaskHandle.js:22]: - [ ] kindly remove unused code from your code base

File: src/index.js [hunk 2]
```diff
@@ -1,22 +1,93 @@
...
+  const todoListArr = JSON.parse(localStorage.getItem('todos') || '[]');
+  const updateList = todoListArr.filter((todo) => todo.completed !== true);
+  updateTodos(updateList);
+};
+
+const removeTodo = (targetIndex) => {
+  const todoListArr = JSON.parse(localStorage.getItem('todos') || '[]');
+  const updateList = todoListArr.filter((todo) => todo.id !== parseInt(targetIndex, 10));
+  updateTodos(updateList);
+};
```
Comment (VICTOR-CHUKWU) [inline src/index.js:42]: - [ ] [Optional] remove todo is not working you might want to check it out :+1:

**Review by VICTOR-CHUKWU — CHANGES_REQUESTED** [review]
Hi @danielochuba  👋  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Good PR
- Good commit
- All linters passed


## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by DammyShittu — APPROVED** [review]
Hi @danielochuba,

## APPROVED 🟢

## Highlights
✔️ Descriptive PR
✔️ Passing linters
✔️ Good branch name

Great implementation of all the requirements in this milestone. 👏 

_You have done an amazing job 👍_

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

Congratulations! 🎉 

## CHEERS AND HAPPY CODING!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**